text/0000-update-version-handling.md

```diff
@@ -0,0 +1,225 @@
+
+- The invocations now allow multiple tool + version combinations, each of the format specified above. For example, a user may simultaneously pin a Node and a Yarn version (with any combination of the version specifier, or none at all):
+
+  ```sh
+  notion install node@10.5 yarn
+  ```
+
+## Explanation for Users
+
+### Current Node users
+
+There are two broad sets of current Node users: those not using a Node version management tool, and those using an existing tool like `nvm`, `nodenv`, or `nvm-windows`.
+
+Current Node users who are not using a Node version management tool should find this a simplification: it matches what npm and Yarn do.
+
+Current Node users using existing tools will find this to be different from their experience:
+
+- `nvm` uses `nodenv install <version>`
+- `nodenv` uses `nodenv install <version>`
+- `nvm-windows` uses `nodenv install <version> [arch]` (where `[arch]` allows users to specify 32-bit or 64-bit builds)
+
+Although we are breaking with these tools, all of them are responsible to manage *only* the Node version, *not* other tools. Notion's purview extends to other tools and therefore overlaps more with the behavior expected from invoking `npm` or `yarn`.
+
+### New Node users
+
+New Node users may have a wide variety of different expectations for adding a version of a tool:
+
+- RubyGems: `gem install <gem name> [-v|--version <version>]`
+
+- Bundler: `bundle add <gem name> [--version=<version>]`
+
+- Nuget: `nuget install <package> [-Version <version>]`
+
+- Paket: `paket add <package> [--version <version spec>]`
+
+- `dotnet`: `dotnet tool install <PACKAGE_NAME> <-g|--global> [--version]`
+
+- Cargo: `cargo install crate [--version <version>]`
+
+- `cargo-edit`: `cargo add <package>[@<version>]...`
+
+Of these, only `cargo-edit` allows adding multiple packages simultaneously; it does so with the same interface as `npm` and `yarn` and proposed here.
+
+# Details
+[details]: #details
+
+## Installing, fetchin, or pinning a single tool
+
+Managing a tool with its default version is unchanged.
+
+| Original                  | Proposed                  |
+| ------------------------- | ------------------------- |
+| `notion install node`     | `notion install node`     |
+| `notion pin yarn`         | `notion pin yarn`         |
+| `notion fetch typescript` | `notion fetch typescript` |
+
+Managing a tool at a specific version, at any level of granularity, now uses `@` to separate the tool name from its version.
+
+| Original                     | Proposed                     |
+| ---------------------------- | ---------------------------- |
+| `notion install node latest` | `notion install node@latest` |
+| `notion pin node 10`         | `notion pin node@10`         |
+| `notion fetch node 10.5`     | `notion fetch node@10.5`     |
+
+This also works for shorthands (currently `latest`, conceivably also tags like `next` or `lts` in the future):
```

Comment:
`lts` is actually already supported (and is actually the default for `notion install node` now), as of this PR: https://github.com/notion-cli/notion/pull/331

Reply:
Yep, I pushed an update with this in the table, but I missed that if mentioned it in the text. Will update. Thanks!